scrapers/src/extractors/office.rs

```diff
@@ -0,0 +1,321 @@
+                r"(?i:District Attorney)",
+                OfficeMeta {
+                    name: "District Attorney".into(),
+                    title: "District Attorney".into(),
+                    r#type: None,
+                    chamber: None,
+                    district_type: Some(db::DistrictType::Judicial),
+                    political_scope: db::PoliticalScope::State,
+                    election_scope: db::ElectionScope::District,
+                },
+            ),
+            (
+                r"(?i:District Court Judge)",
+                OfficeMeta {
+                    name: "District Court Judge".into(),
+                    title: "District Court Judge".into(),
+                    r#type: None,
+                    chamber: None,
+                    district_type: Some(db::DistrictType::Judicial),
+                    political_scope: db::PoliticalScope::State,
+                    election_scope: db::ElectionScope::District,
+                },
+            ),
+            (
+                r"(?i:County Court Judge)",
+                OfficeMeta {
+                    name: "County Court Judge".into(),
+                    title: "County Court Judge".into(),
+                    r#type: None,
+                    chamber: None,
+                    district_type: None,
+                    political_scope: db::PoliticalScope::State,
+                    election_scope: db::ElectionScope::County,
+                },
+            ),
+            (
+                r"(?i:Court of Appeals Judge)",
+                OfficeMeta {
+                    name: "Court of Appeals Judge".into(),
+                    title: "Court of Appeals Judge".into(),
+                    r#type: None,
+                    chamber: None,
+                    district_type: None,
+                    political_scope: db::PoliticalScope::State,
+                    election_scope: db::ElectionScope::State,
+                },
+            ),
+            (
+                r"(?i:Supreme Court Justice)",
+                OfficeMeta {
+                    name: "Supreme Court Justice".into(),
+                    title: "Supreme Court Justice".into(),
+                    r#type: None,
+                    chamber: None,
+                    district_type: None,
+                    political_scope: db::PoliticalScope::State,
+                    election_scope: db::ElectionScope::State,
+                },
+            ),
+            (
+                // TODO - should we have state only extractors? (Colorado RTD)
+                r"(?i:Regional Transportation District Director)",
+                OfficeMeta {
+                    name: "Regional Transportation District Director".into(),
+                    title: "Regional Transportation District Director".into(),
+                    r#type: None,
+                    chamber: None,
+                    district_type: None, // TODO - Add a "transportation" district type ???
```

Comment:
i'm not sure a new district_type makes sense here becaues there isn't a specific district to designate— i think its a denver metro area race so i'm unsure how exactly we will calculate _who_ can vote on this but will do some research and get back to you.

Reply:
> becaues there isn't a specific district to designate— unless there are other tranpsortation districts in the state
For the RTD District Director Office, I'm seeing listings for districts "A", "D", "E", "F", "G", "H', "I", "M". I don't know what these districts map to, I'd guess that RTD has their own arbitrary geographic boundaries based on bus and train routes. I was originally going to put `DistrictType::City` here, but that didn't seem right since RTD goes out to all the Suburbs and Boulder. Idk, open to suggestions here.

Reply:
Found a map
https://www.rtd-denver.com/about-rtd/board-of-directors/board-district-map